pymeasure/instruments/aimtti/aimttiPL.py

```diff
@@ -0,0 +1,176 @@
+        """ Control the output voltage of this channel.
+        With verify: the operation is completed when the parameter being adjusted 
+        reaches the required value to within ±5% or ±10 counts.""",
+        validator=strict_range,
+        values=[0, 6],
+        dynamic=True,
+        get_process=lambda x: float(x[3:])
+        )
+
+    current_limit = Channel.control(
+        "I{ch}?", "I{ch} %g",
+        """ A floating point property that controls the current limit in Amps.""",
+        validator=strict_range,
+        values=[0, 1.5],
+        dynamic=True,
+        get_process=lambda x: float(x[3:])
+        )
+
+    voltage = Channel.measurement(
+        "V{ch}O?",
+        """ Read the output readback voltage for this output channel in Volts.""",
+        get_process=lambda x: float(x[:-1])
+        )
+
+    current = Channel.measurement(
+        "I{ch}O?",
+        """ Read the output readback current for this output channel in Amps.""",
+        get_process=lambda x: float(x[:-1])
+        )
+
+    current_range = Channel.control(
+        "IRANGE{ch}?", "IRANGE{ch} %g",
+        """ A string property that sets the current range of the channel. 
+        Low (500/800mA) range, or High range. 
+        Output must be switched off before changing range.""",
+        validator=strict_discrete_set,
+        values=CURRENT_RANGE,
+        map_values=True
+        )
+
+    def enable(self):
+        """ Enables the output."""
+        self.write("OP{ch} 1")
+
+    def disable(self):
+        """ Disables the output."""
+        self.write("OP{ch} 0")
+
+
+class PLBase(Instrument):
+    """ Control AimTTI PL series power supplies. Model number ending with -P or P(G) support this remote interface.
+
+    Documentation: https://resources.aimtti.com/manuals/New_PL+PL-P_Series_Instruction_Manual-Iss18.pdf
+    PL-series devices: https://www.aimtti.com/product-category/dc-power-supplies/aim-plseries
+
+    .. code-block:: python
+
+        psu = PL303MQDP("ASRL7::INSTR")
+        psu.reset()
+        psu.ch_2.voltage = 1.2
+        psu.ch_2.enable()
+
+    """
+
+    chs = Instrument.ChannelCreator(PLChannel, ("1", "2"), voltage_range=[0, 30], current_range=[0, 3])
+    """ Channels of the power supply. The channels are number from right-to-left, starting at 1. 
+    Default values are for the voltage and current range are for the PL303MQD-P."""
+
+    def __init__(self, adapter, name="AimTTI PL", **kwargs):
+        kwargs.setdefault('timeout', 5000)
+        super().__init__(adapter, name, **kwargs)
+
+    def enable(self):
+        """ Simultaneously sets all outputs on."""
+        self.write("OPALL 1")
+
+    def disable(self):
+        """ Simultaneously sets all outputs off."""
+        self.write("OPALL 0")
+
+    def local(self):
+        """ Go to local."""
+        self.write("LOCAL")
+
+
+class PL068P(PLBase):
+
+    chs = Instrument.ChannelCreator(PLChannel, "1", voltage_range=[0, 6], current_range=[0, 8])
```

Comment:
This has very recently settled down with the release last week that the behaviour was finalised in a different way, in that for instruments with few channels it's clearer/more explicit to add the channels directly: https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments/channels.html#adding-a-channel-with-channelcreator (`ChannelCreator` now does that)
and only use a `MultiChannelCreator` for complex/large configs.

So, here you'd do `ch_1 = Instrument.ChannelCreator(PLChannel, "1", voltage_range=[0, 6], current_range=[0, 8])`